recipes/impactutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7" %}
+
+package:
+  name: impactutils
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/earthquake-impact-utils/archive/{{ version }}.tar.gz
+  sha256: 896081c5e7528c0d5ab11415dcd52037bcc0f5af0756956fa3ef74f68a2b34e1
+
+build:
+  number: 0
+  skip: True
```

Comment:
Just having `skip: True` will skip all builds. If you just want `python >=3.4`  it should be:

```
   ...
   skip: True  # [py27]
```

This will build packages for python 3.5 and 3.6.